python/setup.py

```diff
@@ -194,7 +194,15 @@ def is_pure(self):
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     for i, path in enumerate(LIB_LIST):
         LIB_LIST[i] = os.path.relpath(path, curr_path)
-    setup_kwargs = {"include_package_data": True, "data_files": [("tvm", LIB_LIST)]}
+    LIB_DATA_FILES = []
+    if os.path.isfile(path):
```

Comment:
The variable `path` in `if os.path.isfile(path):` is produced by the `enumerate(LIB_LIST)` above, right? So it seems the indentation of this block doesn't match with the variable of the iteration, and it is only catching the last value on the list? Can you double check on whether this is the expected behaviour?

Reply:
Sorry, it's my mistake and I have fixed it.